README.rst

```diff
@@ -322,11 +328,21 @@ determined by the order of declaration.
 
     --etc
 
-Test with specific params can be called from command line.
+Test from a parameterized group can be called from the command line in such a way:
+
+.. code-block:: Bash
+
+    luatest pgroup.a:1.b:4.test_params
+    luatest pgroup.a:2.b:3.test_params
```

Comment:
`luatest pgroup.b:4.a:1.test_params` will not work?

Reply:
Yes, it will not work. We need to fix this but not in this PR.

Reply:
Okay.